### DISCUSSION TITLE
Grafana 8.1.0 with VictoriaMetrics 1.72.0 help

### DISCUSSION THREAD

**Discussion by kochoudh** [grafana/grafana#46447] [General]
Hello Team,

I need information regarding the compatibility of Grafana 8.1.0 with VictoriaMetrics 1.72.0. I have upgraded Grafana 8.1.0 with VM 1.72.0, however I am not sure of any compatibility issues and was not able to find it anywhere.

Can someone please guide me here.

Regards,
Komal

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
